uvtools/dspec.py

```diff
@@ -60,11 +60,356 @@ def calc_width(filter_size, real_delta, nsamples):
+                                'edge_supression': array-like
+                                    specifies the degree of supression that must occur to tones at the filter edges
+                                    to calculate the number of DPSS terms to fit in each sub-window.
+                                'avg_suppression': list of floats, optional
+                                    specifies the average degree of suppression of tones inside of the filter edges
+                                    to calculate the number of DPSS terms. Similar to edge_supression but instead checks
+                                    the suppression of a since vector with equal contributions from all tones inside of the
+                                    filter width instead of a single tone.
+                            *'clean':
+                                 'tol': float,
+                                    clean tolerance. 1e-9 is standard.
+                                 'maxiter' : int
+                                    maximum number of clean iterations. 100 is standard.
+                                 'pad': int or array-like
+                                    if filt2d is false, just an integer specifing the number of channels
+                                    to pad for CLEAN (sets Fourier interpolation resolution).
+                                    if filt2d is true, specify 2-tuple in both dimensions.
+                                 'filt2d_mode' : string
+                                    if 'rect', clean withing a rectangular region of Fourier space given
+                                    by the intersection of each set of windows.
+                                    if 'plus' only clean the plus-shaped shape along
+                                    zero-delay and fringe rate.
+                                'edgecut_low' : int, number of bins to consider zero-padded at low-side of the FFT axis,
+                                    such that the windowing function smoothly approaches zero. For 2D cleaning, can
+                                    be fed as a tuple specifying edgecut_low for first and second FFT axis.
+                                'edgecut_hi' : int, number of bins to consider zero-padded at high-side of the FFT axis,
+                                    such that the windowing function smoothly approaches zero. For 2D cleaning, can
+                                    be fed as a tuple specifying edgecut_hi for first and second FFT axis.
+                                'add_clean_residual' : bool, if True, adds the CLEAN residual within the CLEAN bounds
+                                    in fourier space to the CLEAN model. Note that the residual actually returned is
+                                    not the CLEAN residual, but the residual in input data space.
+                                'taper' : window function for filtering applied to the filtered axis.
```

Comment:
although `taper` is used in hera_pspec, `window` is the kwarg we have been using here. not sure if this change was accidental or not.

Reply:
Technically this was an accident. Not sure which makes more sense here. Probably window. I've changed instances of taper to window.